arbos/programs/native.go

```diff
@@ -93,6 +93,7 @@ func activateProgram(
 func callProgram(
 	address common.Address,
 	program Program,
```

Comment:
This `program` parameter is no longer used now that `moduleHash` is passed in separately.